app/controllers/ConfirmationController.scala

```diff
@@ -101,11 +101,14 @@ class ConfirmationController @Inject()(appConfig: FrontendAppConfig,
     ))
   }
 
-  private def getCr03(reportStatus: ReportStatus): Option[Cr03Submission] = {
+  private def getCr01Cr03(reportStatus: ReportStatus): Option[Cr01Cr03Submission] = {
     reportStatus.report
       .map(_.value)
-      .filter(x => x.get("type").map {case x: JsString => x.value == "Cr03Submission"}.getOrElse(false))
-      .flatMap(x => x.get("submission")).flatMap(x => Cr03Submission.format.reads(x).asOpt)
+      .filter(x => x.get("type").exists {
+        case x: JsString => x.value == "Cr01Cr03Submission"
```

Comment:
So backend and frontend will always use `Cr01Cr03Submission` and backend have compatibility for previous `Cr03Submission` which can me in database.

Reply:
Yes - this is the line for backward compatibility https://github.com/hmrc/voa-bar/blob/master/app/uk/gov/hmrc/voabar/services/WebBarsService.scala#L74